mlflow/pyfunc/backend.py

```diff
@@ -64,11 +64,12 @@ def serve(self, model_uri, port, host):
         # NB: Absolute windows paths do not work with mlflow apis, use file uri to ensure
         # platform compatibility.
         local_uri = path_to_local_file_uri(local_path)
-        command = ("gunicorn --timeout 60 -b {host}:{port} -w {nworkers} "
+        command = ("gunicorn --timeout 60 -b {host}:{port} -w {nworkers} ${{GUNICORN_CMD_ARGS}}"
```

Comment:
@LarsDu I think we still need a space here after `${{GUNICORN_CMD_ARGS}}`

Reply:
You are correct. Somehow after adding the space, I still get the following error:

```
Traceback (most recent call last):
  File "/Users/larryd/larryd_osx_venv/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/Users/larryd/larryd_osx_venv/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/Users/larryd/larryd_osx_venv/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/Users/larryd/larryd_osx_venv/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/Users/larryd/larryd_osx_venv/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/Users/larryd/larryd_osx_venv/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/Users/larryd/larryd_osx_venv/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
ModuleNotFoundError: No module named '${GUNICORN_CMD_ARGS}'
```

This time I cannot figure out why.
I think I will just bite the bullet and do an os.environ.get('GUNICORN_CMD_ARGS')